tests/Resources/Page/OCPDashboard/Monitoring/Metrics.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Library  SeleniumLibrary
+
+*** Variables ***
+${METRICS_QUERY_TEXTAREA}  xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div/div[1]/textarea
+${METRICS_QUERY_RESULTS_PARENT_ELEMENT}            xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]
+${METRICS_QUERY_RESULTS_TABLE}                     xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]/table
+${METRICS_QUERY_RESULTS_TABLE_ROW1_VALUE_ELEMENT}  xpath=/html/body/div[2]/div[1]/div/div/div/div/main/div/div/div/section/div/div[2]/div[2]/div[2]/div/div[3]/div[2]/table/tbody/tr/td[4]
```

Comment:
This is likely to break with the smallest change to the page structure. Is there any way to simplify this where you search for a parent div by title, class or id?

Reply:
I've simplified the locators in today's commit